lib/puppet/transaction/report.rb

```diff
@@ -183,7 +194,7 @@ def initialize(kind, configuration_version=nil, environment=nil, transaction_uui
     @host = Puppet[:node_name_value]
     @time = Time.now
     @kind = kind
-    @report_format = 5
+    @report_format = 6
```

Comment:
We talked about this some with @nfagerlund last time around, and the policy (I think) we converged on was that we would bump it for any new (or removed) field, but not more often than once per release. (E.g. if we add another change before 4.6 we wouldn't need to bump the report format number *again*.)

I think the reasoning here was that the report format gives a consumer one place to look at to know what fields to expect (and bumping it has very little (any?) cost).

Nick, does that match your memory?